cmd/gke-exec-auth-plugin/tpm.go

```diff
@@ -114,19 +114,22 @@ func tpmAttest(dev tpmDevice, privateKey crypto.PrivateKey) ([]byte, error) {
 	}
 	glog.Info("certification signature verified with AIK public key")
 
+	// Try loading AIK cert, but don't fail if it wasn't provisioned.
```

Comment:
We'll eventually want to remove this right?

Reply:
Yes. Added a TODO below